Zend/zend_alloc.c

```diff
@@ -454,9 +454,13 @@ static void *zend_mm_mmap(size_t size)
 #ifdef _WIN32
 	void *ptr;
 #ifdef _WIN64
+	static zend_long minlgsz = -1;
```

Comment:
A static variable won't play nicely with thread-safe builds. I think we need to initialize the variable in `start_memory_manager()`, so it won't be modified after startup.